best-practices/setting_up_rails_projects.md

```diff
@@ -0,0 +1,23 @@
+# Setting up Rails Projects
+
+Many of the DLSS software projects are [Ruby on Rails](http://rubyonrails.org/) applications. Consistency across projects can aid developers and reduce the overall maintenance burden on teams. This guide should be used as an outline on common expectations in DLSS Ruby on Rails projects. Not only should our projects reflect consistency internally, but should also reflect larger community expectations. These are not to the letter rules, but guidelines aimed to help the team.
```

Comment:
Drop "to the letter?" It parses weird to me, and I don't think it adds much to your (very good) point.